src/ConnectedMode/Migration/IUnintrusiveBindingController.cs

```diff
@@ -0,0 +1,36 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.ComponentModel.Composition;
+
+namespace SonarLint.VisualStudio.ConnectedMode.Migration
+{
+    internal interface IUnintrusiveBindingController
+    {
+        void Bind();
+    }
+
+    [Export(typeof(IUnintrusiveBindingController))]
+    [PartCreationPolicy(CreationPolicy.Shared)]
+    public class UnintrusiveBindingController : IUnintrusiveBindingController
+    {
+        public void Bind() { }
```

Comment:
You could have, but it doesn't matter too much since we know we're going add the implementation shortly.

Normally, I'd add a TODO comment referencing the ticket that relates to creating the implementation (which in turn creates another code smell since it's a TODO. However, from a reader point of view it's clear that there is work planned to remove the TODO).